core/src/test/java/nu/yona/server/exceptions/ResourceBasedExceptionTest.java

```diff
@@ -43,7 +40,7 @@ protected TestException(String messageId, Object... parameters)
 
 	}
 
-	@Test
+	// @Test
```

Comment:
I just pushed an update to your branch that I expect to fix the issue. Please check.